src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,111 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int capasitu = 16;
```

Comment:
The variable name 'capasitu' is not informative and contains a typo. It's better to use a more common name like 'capacity'. Remember that variable names should be written in English, be informative and they should follow Java naming conventions.